pkg/operator/csidriveroperator/csioperatorclient/gcp-pd.go

```diff
@@ -0,0 +1,39 @@
+package csioperatorclient
+
+import (
+	"os"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+)
+
+const (
+	GCPPDCSIDriverName          = "pd.csi.storage.gke.io"
+	envGCPPDDriverOperatorImage = "GCP_PD_DRIVER_OPERATOR_IMAGE"
+	envGCPPDDriverImage         = "GCP_PD_DRIVER_IMAGE"
+)
+
+func GetGCPPDCSIOperatorConfig() CSIOperatorConfig {
+	pairs := []string{
+		"${OPERATOR_IMAGE}", os.Getenv(envGCPPDDriverOperatorImage),
+		"${DRIVER_IMAGE}", os.Getenv(envGCPPDDriverImage),
```

Comment:
These env vars aren't set anywhere else in this PR, are they?

You should specify them in `manifests/09_deployment.yaml`

Reply:
Looking at the other operators I don't see any of these doing a similar method? They're all specified in the corresponding *.go files?

Reply:
I'm not sure I understand your question...

At lines 18 and 19 the code is reading `GCP_PD_DRIVER_OPERATOR_IMAGE` and `GCP_PD_DRIVER_IMAGE` env vars. However, they are not set anywhere, so they are going to be empty.

We should set them in the CSO deployment, like we do for AWS for example: https://github.com/openshift/cluster-storage-operator/blob/master/manifests/09_deployment.yaml#L50-L53

Reply:
I misunderstood what you were saying. I'll add these variables once they're available on quay.

From speaking with Thiago this should happen soon.